src/lib/connectors/water-import/jobs.js

```diff
@@ -1,16 +1,48 @@
 'use strict';
 
 const { pool } = require('../db');
+const { get } = require('lodash');
+const { pgBossJobOverview } = require('./queries');
 
-const getJobSummaryQuery = `
-  select name, state, count(*)
-  from water_import.job
-  group by name, state;
-`;
+const getJobSummary = () => {
+  /**
+   * The purpose of this route is to check the state of import jobs.
+   *
+   * The checks are intentionally broken into separate queries. This is because the import
+   * microservice is currently powered by PGBoss while other parts of The Service are powered
+   * by BullMQ. The assumption/anticipation is that at some point, import jobs will coexist across
+   * multiple job handlers... Maybe.
+   */
+  const pgBossJobsArray = [
+    { id: 'nald-import.import-licence', displayName: 'Licences (NALD)' },
+    { id: 'import.licences', displayName: 'Licences' },
+    { id: 'import.licence', displayName: 'Licence' },
+    { id: 'import.delete-documents', displayName: 'Delete removed documents' },
+    { id: 'nald-import.delete-removed-documents', displayName: 'Delete removed documents (NALD)' },
+    { id: 'nald-import.s3-download', displayName: 'NALD Zip Download' },
+    { id: 'nald-import.populate-pending-import', displayName: 'Populate pending import' },
+    { id: 'import.bill-runs', displayName: 'Bill runs' },
+    { id: 'import.charging-data', displayName: 'Charging data' },
+    { id: 'import.companies', displayName: 'Companies' },
+    { id: 'import.company', displayName: 'Company' }
+  ];
 
-const getJobSummary = async () => {
-  const result = await pool.query(getJobSummaryQuery);
-  return result.rows;
+  return Promise.all(pgBossJobsArray.map(async eachJob => {
+    const { rows: status } = await pool.query(pgBossJobOverview, [eachJob.id]);
+
+    const failedCount = parseInt(get(status.find(row => row.state === 'failed'), 'count', 0));
+    const completedCount = parseInt(get(status.find(row => row.state === 'completed'), 'count', 0));
+    const active = !!status.find(row => row.state === 'active') || !!status.find(row => row.state === 'created');
+    const lastUpdated = get(status.find(row => row.state === 'completed'), 'max_completed_date', null);
+
+    return {
+      displayName: eachJob.displayName,
+      failedCount,
+      completedCount,
+      active,
```

Comment:
We have a general pattern of naming boolean fields as `isSomething` as a prefix in the service, e.g.:
`isActive`